server/routes/allocation.js

```diff
@@ -78,4 +87,35 @@ router.patch(
   })
 );
 
+// Create bulk allocation: POST
```

Comment:
At some point we might wanna roll this into the main allocation route, kind of like what we discussed with the other upcoming bulk work. This is fine for now though.

Reply:
That was mentioned in the dev channel - I did move it OOS at this point due to the logic of post vs patch whereas the example we discussed only involved post. If we merge this feature into the existing POST, some updates/decisions will need to be made regarding the existing UI. (ie if for some reason they try to post a new allocation and one exists, do we allow/override and inform the user, or prevent that from occurring as it currently is implemented). 
If there is time after testing is completed I can look into merging the two.